scripts/00_setup.R

```diff
@@ -1,6 +1,8 @@
 install.packages("groundhog", repos="https://ftp.fau.de/cran/")
 library("groundhog")
+# TODO: setwd() for this R session to scripts/? Or ask users to setwd() accordingly, otherwise groundhog & paths don't work as expected.
 set.groundhog.folder("./R_groundhog/")
+# TODO: cmdstanr is required as well, if not for the priors then for the models. Mention either here or in 03/04/05?
```

Comment:
If you'd like @Tarotis I've written a script that checks which functions are used in all R-scripts and compares to what is loaded. It struggles with some complex package-structures, but I think it's a good sanity check.
https://github.com/HedvigS/Oceanic_computational_ASR/blob/main/code/01_check_used_functions.R

Reply:
Should be complete now, but I will remember it for future projects, thanks!